Week2/homework/maartjes-work.js

```diff
@@ -47,15 +47,23 @@ const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
```

Comment:
Nicely done in general, you might have noticed that assigning the result of each `map` or `filter` operation to a variable is repetitive and in this case probably not very useful since you never use those variables.
You could use the chaining property of these function and rewrite everything by chaining `map` and `filter` calls